stm/coqworkmgrApi.ml

```diff
@@ -8,8 +8,18 @@
 
 let debug = false
 
+type priority = Low | High
+let string_of_priority = function Low -> "low" | High -> "high"
+let priority_of_string = function
+  | "low" -> Low
+  | "high" -> High
+  | _ -> raise (Invalid_argument "priority_of_string")
+
+let async_proofs_n_workers = ref 1
```

Comment:
IIRC my main requrest was to remove these two flags from here, since they don't belong here (they are just used as defaults here).  The place for these is the STM proper (stm.ml for example, where they are used to initialize the worker queues).

Reply:
Can you elaborate on how then `CoqworkmgrAPI` is supposed to work. I am failing to make sense of the `min` stuff.

Reply:
You want an additional parameter to `get` then ?

Reply:
Well it doesn't matter anyways, `get` is always called with 1, so indeed the parameters can be just removed I think.